cmd/clusterctl/internal/test/contracts.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
+
+// PreviousCAPIContractNotSupported define the previous Cluster API contract, not supported by this release of clusterctl.
+const PreviousCAPIContractNotSupported = "v1alpha3"
+
+// CurrentCAPIContract define the current Cluster API contract.
+var CurrentCAPIContract = clusterv1.GroupVersion.Version
+
+// NextCAPIContractNotSupported define the next Cluster API contract, not supported by this release of clusterctl.
+const NextCAPIContractNotSupported = "v1alpha5"
```

Comment:
Is there a better way to convey these constants? It'd be great to find a different solution, rather than hardcoding the API version. Could we leverage the core provider metadata?

Reply:
We can pick up v1alpha3 from the old API
For valpha5, we can increase the latest number in CurrentCAPIContract, but I'm not sure how robust it will be, opinions?

The nice thing is that those values are defined only here, so it should not be a big burden to maintain them every time a release is bumped

Reply:
fixed, PTAL